harithakarma/lib/Screens/Homeuser/HomeSideDrawer.dart

```diff
@@ -0,0 +1,141 @@
+import 'package:flutter/material.dart';
+import 'package:harithakarma/Screens/Auth/login.dart';
+import 'package:harithakarma/Screens/Homeuser/HomeUserHistory.dart';
+import 'package:harithakarma/Screens/Homeuser/HomeUserHome.dart';
+import 'package:harithakarma/Screens/Homeuser/HomeUserNotifications.dart';
+import 'package:harithakarma/Screens/Homeuser/HomeUserOrder.dart';
+import 'package:harithakarma/Screens/Homeuser/complaints.dart';
+import 'package:harithakarma/Screens/Homeuser/profile.dart';
+import 'package:harithakarma/Screens/Homeuser/settings.dart';
+import 'package:harithakarma/service/auth.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+class HomeSideDrawer extends StatefulWidget {
+  const HomeSideDrawer({Key? key}) : super(key: key);
+
+  @override
+  State<HomeSideDrawer> createState() => _HomeSideDrawerState();
+}
+
+class _HomeSideDrawerState extends State<HomeSideDrawer> {
+  @override
+  Widget build(BuildContext context) {
+    return Drawer(
+      child: Column(
+        children: <Widget>[
+          const DrawerHeader(
+            decoration: BoxDecoration(
+              color: Color.fromARGB(255, 23, 75, 7),
+            ),
+            child: Center(
+              child: Text(
+                'HarithaKarma',
+                textAlign: TextAlign.center,
+                style: TextStyle(color: Colors.white, fontSize: 25),
+              ),
+            ),
+          ),
+          ListTile(
+            leading: const Icon(Icons.home),
+            title: const Text('Home'),
+            onTap: () => {
+              Navigator.push(
```

Comment:
this will give user to push `HomeUserHome()`  many times on top of existing window